packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1432,11 +1432,15 @@ module.exports.external = [
             const paragraphArray = Array.isArray(td.paragraph) ? td.paragraph : [td.paragraph];
             const textValues = paragraphArray
               .map(paragraph => {
-                if (!paragraph || !paragraph.content) return "";
+                if (!paragraph || (!paragraph.content && !paragraph._)) return "";
```

Comment:
Sure this is what we want? Return "" only if the paragraph is not set or if both `content` and `_` are not set?

Just to make sure we don't want just another logical or.